profile_collection/startup/99-subtracted-tiff-exporter.py

```diff
@@ -51,5 +19,9 @@ def event(self, doc):
         doc['data'][self.field] = subtracted_img
         super().event(doc)
 
-template = "/home/xf28id1/xpdUser/tiff_base/LaB6_EPICS/{start.sa_name}_{start.scan_id}_step{event.seq_num}.tif"
+
+# TODO this looks too hard-coded!
+template = ("/home/xf28id1/xpdUser/tiff_base/LaB6_EPICS/"
```

Comment:
I don't think we will need this, since a) I am fairly certain that we don't save files with `LaB6_EPICS` in the name and b) we will be handling this on the pipeline side (I hope!)

Reply:
If you now handle export someplace else, just delete it this whole file.

Reply:
@chiahaoliu thoughts?

Reply:
The tiff-exporter I wrote [a while ago](https://github.com/xpdAcq/xpdAn/blob/master/xpdan/callbacks_core.py#L18) is essentially a derivative of this subtracted-tiff-exporter, with some tweaks on file name template so that it matches outputs of ``save_last_tiff``. Maybe we could PR into here? Just for a backup.

Reply:
:-1: to back up copies of implementations, it is easy to lose track of which one is the 'real' one.

Reply:
:+1: sounds fair.